README.md

```diff
@@ -212,6 +212,13 @@ You can even run python scripts like this:
   shell: pixi run python {0}
 ```
 
+Using Windows? Try `powershell` or `bash`:
+```yml
+- run: | # everything here will be run inside of the pixi environment
+    python --version | Select-String "3.11"
+  shell: pixi run pwsh -Command {0} # pwsh works on all platforms
```

Comment:
I'm wondering why this even works 🤔 shouldn't it be `-File`? The thing in `{0}` will be replaced by a path to the script at runtime...

![image](https://github.com/prefix-dev/setup-pixi/assets/29506042/e7ec8cbe-8861-4382-9a4d-d90c1783a8c2)

Reply:
I started with `-File` but changed to `-Command` because that is what github uses: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
> GitHub appends the extension .ps1 to your script name.

interesting... we unfortunately don't have the power to do this :(

Reply:
let's also change the bash shell to `bash -e` while we're at it

Reply:
I also find it super weird that there is different bash behavior when specifying `bash` or not specifying anything on unix...

![image](https://github.com/prefix-dev/setup-pixi/assets/29506042/b6d8cf00-1ec5-4278-9fd3-2c70c28daeed)

Reply:
yeah no clue this whole shell thing isn't the most ergonomic feature.